### DISCUSSION TITLE
自带拼音方案的拼音问题

### DISCUSSION THREAD

**Discussion by SogaSuga** [rime/home#835] [General]
一个贴子 https://tieba.baidu.com/p/2757690418?red_tag=0327150490
其13楼提到：

> 由於rime单字的读音是为传统汉字优化的，所以部分借用的传统汉字字形的简化字词语，自动注音后会有错。
如「叶」字，自古以来是「协」的异躰字，其读音一直以来只有一个「xie2」，其组词有「叶音」、「叶韵」（这两个词都是音韵学的术语）、「叶恭弘」（一个日本漫画家）等等。
而「叶」作「树枼」的「枼」的简化字讲，最早见於民国时期江淮地区包点店的师傅将「枼」写成「叶」别字所致，就頪似於今天的餐庁菜单上「豆腐」的别字写法「豆付」、「蛋炒饭」的别字写法「旦炒饭」一様。
所以rime的自动注音的时候，只会给帯「叶」字的词彚注上「xie2」音，而「ye4」当成罕用音不会参与自动注音。而「叶」（ye4）在简化字词语中又是一个常用字，所以把简化字词语交给经过传统汉字字音优化的rime来処理，会彂生很多意想不到的错误。

实测确实如此，比如自建一个词条“叶条条”，不加拼音码，需要打 xie tiao tiao 才出来。
虽然平时几乎无法感知到有问题，但还是想问问有没有什么好的，自动的解决方法？
是不是要自行修改各方案的基础词库？

贴吧内容不知道啥时候就可能消失，所以截图备份如下，第3点：
<img width="884" alt="71AFAE45-79EC-4D77-8D8B-5FAE9F9EB674" src="https://user-images.githubusercontent.com/66196437/125732028-9d180cf7-a971-4963-bdcd-2840da5a89c9.png">

**Comment by LEOYoon-Tsaw** [comment]
建「葉條條」，而不是「叶條條」。如果非要用簡體詞庫，就簡體做全套。

**Reply by SogaSuga** [reply]
明白了
